recommenders/__init__.py

```diff
@@ -24,10 +24,6 @@
 
 
 
-
-movies_df = pd.read_csv("movies.csv")

```

Comment:
The commented out code on line 5 is unnecessary and should be removed to improve readability.